vertical-pod-autoscaler/enhancements/XXXX-restore-selector-support.md

```diff
@@ -0,0 +1,105 @@
+# AEP-XXXX: Restore Label Selector Support to VPA
```

Comment:
I think the AEP title should change to reflect that this proposal aims to improve or solve heterogeneous workload scaling in VPA for `Deployments` and `StatefulSets`. Restoring label selector support is only one part of the solution.

so please make the intention of the AEP in its title more explicit, thanks!